### PR TITLE
Fixed bugs introduced by changing variable naming format

### PR DESCRIPTION
**Pull request by X3Technology**
This PR fixes many of the bugs introduced by changing the format of field names and variable names.

#60

### CODE REVIEW

File: DTOs/ContentFieldDto.cs [hunk 1]
```diff
@@ -38,6 +38,6 @@ public class ContentFieldDto
 
         public ContentFieldTypeDto ContentFieldType { get; set; }
 
-        public object Value { get; set; }
+        //public object Value { get; set; }
```
Comment (valadas) [inline DTOs/ContentFieldDto.cs]:
```suggestion
```

I am not a big fan of commented out code.

File: DTOs/RelationshipDtoExtensions.cs [hunk 2]
```diff
@@ -35,6 +35,8 @@ public static RelationshipDto ToDto(this StructuredContent_Relationship item)
                 BContentType = item.StructuredContent_ContentType1?.ToDto(),
             };
 
+           
+
```
Comment (valadas) [inline DTOs/RelationshipDtoExtensions.cs]:
```suggestion
```

Stylecop will probably bark at more than a single blank like

File: app/content-field-types/related-content-single/controller.js [hunk 3]
```diff
@@ -1,62 +1,33 @@
-﻿app.controller('contentFieldTypeRelatedContentSingleController', ['$scope', '$q', 'toastr', 'contentTypeService', 'contentItemService', function ($scope, $q, toastr, contentTypeService, contentItemService) {
+﻿app.controller('contentFieldTypeRelatedContentSingleController', ['$scope', '$q', 'toastr', 'contentItemService', function ($scope, $q, toastr, contentItemService) {
 
     $scope.relationship;
-    $scope.primaryContentType;
-    $scope.relatedContentType;
+
+    //$scope.primaryContentType;
+    //$scope.relatedContentType;
```
Comment (valadas) [inline app/content-field-types/related-content-single/controller.js]:
```suggestion
```

I am not a big fan of commented out code

File: app/content-items/edit/controller.js [hunk 4]
```diff
@@ -92,7 +87,7 @@
             $scope.contentItem.status = "Draft";
             $scope.contentItem.datePublished = null;
 
-            prepItemForSave();
+            //prepItemForSave();
```
Comment (valadas) [inline app/content-items/edit/controller.js]:
```suggestion
```
I am not a big fan of commented out code

File: app/content-items/edit/controller.js [hunk 5]
```diff
@@ -130,7 +125,7 @@
 
             $scope.contentItem.status = "Published";
 
-            prepItemForSave();
+            //prepItemForSave();
```
Comment (valadas) [inline app/content-items/edit/controller.js]:
```suggestion
```
I am not a big fan of commented out code

File: app/content-types/edit/controller.js [hunk 6]
```diff
@@ -25,6 +24,7 @@
         $scope.loading = true;
         contentFieldTypeService.search("", true).then(
             function (response) {
+                console.log('getContentFieldTypes', response.data);
```
Comment (valadas) [inline app/content-types/edit/controller.js]:
```suggestion
```

console logging should probably not get committed into production code.

File: app/content-types/edit/controller.js [hunk 7]
```diff
@@ -44,6 +44,7 @@
 
         contentFieldTypeService.get($scope.contentField.contentFieldTypeId).then(
             function (response) {
+                console.log('getContentFieldType', response.data);
```
Comment (valadas) [inline app/content-types/edit/controller.js]:
```suggestion
```

console logging should probably not get committed into production code.

File: app/content-types/edit/controller.js [hunk 8]
```diff
@@ -64,10 +65,10 @@
 
         contentTypeService.get($scope.contentType.id).then(
             function (response) {
+                console.log('getContentType', response.data);
```
Comment (valadas) [inline app/content-types/edit/controller.js]:
```suggestion
```

console logging should probably not get committed into production code.

File: app/content-types/edit/controller.js [hunk 9]
```diff
@@ -184,6 +147,7 @@
 
         contentFieldService.search($scope.contentType.urlSlug, true).then(
             function (response) {
+                console.log('getContentFields', response.data);
```
Comment (valadas) [inline app/content-types/edit/controller.js]:
```suggestion
```

console logging should probably not get committed into production code.

File: app/content-types/edit/controller.js [hunk 10]
```diff
@@ -257,6 +221,7 @@
 
         relationshipService.search($scope.contentType.id).then(
             function (response) {
+                console.log('getRelationships', response.data);
```
Comment (valadas) [inline app/content-types/edit/controller.js]:
```suggestion
```

console logging should probably not get committed into production code.

**Review by valadas — APPROVED** [review]
I see nothing wrong but offered some suggestions.

**Comment by X3Technology** [comment]
Good catches.  I'll clean it up some more tomorrow.

**Comment by valadas** [comment]
Awesome, merging, thanks!

**Pull request merged**